### PR TITLE
Move to use lerna to have multiple packages inside same repo

### PR DESCRIPTION
**Pull request by sveisvei**
Will close https://github.com/finn-no/unleash/issues/135.

This is a move to use Lerna for managing multiple packages.
It is not production ready, but part of moving and splitting up into own modules.

Currently it is green, and should be merged imo. #136 also does split the docker config into own package, therefor I did not touch it.

* I removed linting as part of testing now, as I will do this in an own PR with fixes and updating to node 6 /ES 6.

**ps: semaphoreci wont build because this changes the setup, and we need to reconfigure it after merge.**

### CODE REVIEW

File: packages/unleash-server/package.json [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    "jest-cli": "0.5.8",
+    "mocha": "^2.4.5",
+    "mocha-lcov-reporter": "1.2.0",
+    "nsp": "^2.3.2",
+    "pre-commit": "^1.0.2",
+    "react-tools": "^0.13.1",
+    "supertest": "^1.2.0",
+    "supervisor": "^0.10.0"
+  }
+}
```
Comment (ivarconr) [inline packages/unleash-server/package.json]: this probably needs to depend on `unleash-frontend`?

Comment (sveisvei) [inline packages/unleash-server/package.json]: will fix, not done this one yet.

**Comment by sveisvei** [comment]
@ivarconr Could we run with an unstable master for a while?

**Comment by ivarconr** [comment]
I guess if we make a 1.0.0 branch and release from that, I see no reason not to. It also depends on your definition of "a while" i guess :palm_tree:

**Comment by sveisvei** [comment]
Well, after these changes master should not be "production ready" between releases, we releases for that right. Works with an branch, or just start from the release-git-tag. 👍

**Comment by ivarconr** [comment]
true, a small link to a downloadable version could be helpful inside the readme.

File: .eslintrc [hunk 2]
```diff
@@ -1,24 +1,5 @@
...
-        "dot-notation": 0,
-        "eqeqeq": 0,
-        "no-eq-null": 0,
-        "no-undefined": 0,
-        "no-else-return": 0,
-        "no-shadow": 0,
-        "consistent-this": 0
-    }
+        "finn"
+    ]
```
Comment (ivarconr) [inline .eslintrc]: be careful and try not to solve all issues in one PR!

**Comment by ivarconr** [comment]
It will be good to have this merged. Can you please write a small paragraph in the README.md on how a developer should publish a new version of unleash after lerna is introduces?

**Comment by sveisvei** [comment]
@ivarconr sure thing, will add readme and remove eslint stuff.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # unleash
 
+__Warning: We are in the process of splitting up unleash into multiple packages in this repository, if you want to test the previous package see [previous tag](https://github.com/finn-no/unleash/tree/v1.0.0-alpha.1) __
```
Comment (ivarconr) [inline README.md]: i actually released v1.0.0-alpha.2 yesterday.

Comment (sveisvei) [inline README.md]: ok, but its partly packages - so this is from the state before splitting.

Btw, Im rethinking if its best to release 1.0.0....

**Pull request merged**